pages/Styles-Overview.md

```diff
@@ -13,10 +13,13 @@ The `polygons` draw style tessellates and extrudes vector shapes into 3D geometr
 The `lines` draw style can turn either polygonal or line data into lines.
 
 #### `points`
-The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the `[priority](draw.md#priority)` draw parameter.
+The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the [`priority`](draw.md#priority) draw parameter.
 
 #### `text`
-The `text` draw style draws a rectangle at a point, and paints it with a texture it generates automatically based on its input datasource. It can work with point, line, or polygon data. Text labels will "collide" with each other, with only the winner being drawn, determined by the `[priority](draw.md#priority)` draw parameter.
+The `text` draw style draws a rectangle at a point, and paints it with a texture it generates automatically based on its input datasource. It can work with point, line, or polygon data. Text labels will "collide" with each other, with only the winner being drawn, determined by the [`priority`](draw.md#priority) draw parameter.
+
+#### `raster`
+The `raster` draw style draws one tile-sized square per tile and paints it with the appropriate tile from a `Raster` data source.
```

Comment:
Similarly seems like this could be simplified a bit to just say it renders raster tiles in the traditional slippy map fashion? (I struggle with explaining this in plain terms too though)